e2e/cli/command/run.go

```diff
@@ -0,0 +1,70 @@
+package command
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+func RunCommandFactory() (cli.Command, error) {
```

Comment:
Mostly since it makes sense with this context: https://github.com/hashicorp/nomad/blob/5bcdf75d1b7a278ae4051548f1099b1715d5e18a/e2e/cli/main.go#L16-L19

Its less of a constructor and more of a type implementation? I'm not opinionated about it just describing the reasoning.